Main.hs

```diff
@@ -39,7 +39,6 @@ import DupQueue
 
 getAuth :: IO Auth
 getAuth = Auth <$> read <$> readFile "auth.conf"
-login = loginRest =<< getAuth
```

Comment:
This line was needed so that I could do maintenance from GHCi.

Reply:
Ahh that makes a lot of sense. I've removed this change from the pull request.